app/styles/app/modules/buttons.scss

```
@@ -233,7 +233,8 @@
 
 .action-button--restart,
 .action-button--debug,
-.action-button--cancel {
+.action-button--cancel,
+.action-button--priortize {
```

There's a typo in the class name:
```suggestion
.action-button--prioritize {
```

---

Done.